src/js/internal/sql/shared.ts

```diff
@@ -441,6 +456,7 @@ function parseOptions(
     options.user ||
     env.PGUSERNAME ||
     env.PGUSER ||
+    env.MYSQL_USER ||
```

Comment:
what if the user chose `adapter: "mysql"` but they didn't specify a username explicitly and they have `PGUSER` in an environment variable?